docs/charter.md

```diff
@@ -19,9 +19,14 @@ This charter lays out the background, intentions, mandate, scope, means and meth
 This document also explicitly acknowledges a few related organizations and communitites, and spell out responsibility demarcation lines between these, if any.
 
 
+## Name
+
+The full name of the group is **CPAN Security Group**. Valid abbreviations are **CPANSec**.
```

Comment:
I may have missed a discussion or two on irc. Shouldn't it be "Working Group" instead of just "Group"?

Reply:
I'd suggest changing this line to "The full name of the group is **CPAN Security Working Group**, abbreviated as **CPANSec**".

Reply:
The name change came up at the last video meeting we had. We voted, 2 in favor, 4 abstaining. :slightly_smiling_face:

Reply:
Oh, and if you think this was too rashly done (or not done in a proper manner), then please send an agenda item addressing this to @timlegge (he's the meeting secretary next this week).

Reply:
I thought I didn't but I think I do, maybe? 

I mean, "working group" is a very well-known term with a very specific definition. From Oxford Languages we have _"a committee or group appointed to study and report on a particular question and make recommendations based on its findings._" From WIkipedia we have _"a group of experts working together to achieve specified goals. The groups are domain-specific and focus on discussion or activity around a specific subject area."_

Whereas a group is just _"a number of people or things close together"._

I also think a name change should have happened with a little more active participation of the community instead of being decided top-down on a single meeting. Like when we settled for "CPANSec", where we had a long talk over on irc, tagging people, waiting a couple of days for their input, etc. Or maybe you had that and I just missed it. Oh, well.

Unfortunately I won't be able to attend the online meetings until maybe after the PTS so unless what I expressed here resonated with you all on some level, it's a done deal and so be it. But I as still curious as to why it was a subject of discussion and vote on the last video meeting.

Reply:
Either way, this looks like a conversation for irc or meetings. I'll just be satisfied that it was not a typo and move on.

Reply:
Happy to admit that this happened quickly (maybe even too quick!), and we would have done well if we had put the proposal forward on the chat, in time for more of us to absorb and consider any ramifications.

Another amber flag is that when we voted, we had two voting in favor and four abstaining. Maybe we should have a requirement that votes need a true majority?

We don't really have established any formal decision process, and I'm thinking this case may be a reason for us to do exactly that.

@garu: would you be ok putting forward a short proposal? Either as a point in the agenda (under "operating changes", which is the topic where we improve how meetings are conducted), or as a suggestion in the ticket we have specifically for this topic: https://github.com/CPAN-Security/security.metacpan.org/issues/19

Reply:
On the other side of this issue, we _do_ need people to actually participate and be available when decisions are being made.

When someone is not available, this is functionally equivalent to not being part of the project – and therefore has the same consequences on the health and progress of the project.

No project thrives solely on it's bystanders.

@garu, I think I've asked both you and @petek to find some meeting time when you guys _can_ be available. As far as I can tell, none of you have offered any alternative meeting hours (if I'm wrong here, please correct me). I've also asked @ingy the same, and he urged us to move the meeting two hours, which we did.

So in practice (when it comes to discussing and making decisions) we have too many among us who are functionally indistinguishable from any bystander. How would you improve on this situation?

Reply:
@sjn But I did, and I did it the very first time you asked. it was just never acknowledged =/ 

```
[2024-01-19 06:09] <@sjn> Remember the meeting on slack, tomorrow at 12:00
UTC (we'll vote on moving it to another time/day then, for those of you who
would like that.)

[2024-01-19 06:09] <@sjn> garu: do you have a proposal about change of
meeting day/time for the meeting tomorrow? ^

(...)

[2024-01-19 19:56] <garu> sjn: any weekday works for me but it looks
like wednesdays would be good for a lot of people here. I'm in UTC-3
and usually can connect anytime except between 11am and 2pm (when
the kid eats), and 7pm-9pm (when the kid sleep)

[2024-01-19 19:59] <garu> sleeps, even. So anytime except 14:00-17:00
UTC and 22:00-00:00 UTC. I think I can squeeze it at 16:00 UTC but I may not
be able to make it every day at that time, depends on traffic (taking the
kid to school after lunch)

[2024-01-19  20:44] <@sjn> garu: I'm happy to accept a proposal from
you. Pick a time! :)

[2024-01-19 21:51] <garu> sjn: how about wednesdays at 12:00 or 13:00 UTC?
```